File: calculadora.py
lista = ["+", "-", "*", "/", "%", "//", "**"]
def calculo(opção):
    if opção == 1:
        operacao = input('''
escolha a operacao
+ para adicao
- para subtracao
* para multiplicacao
/ para divisao
% para modulo
// para divisao minima
** para exponencial
''')
        
        while operacao not in lista:
            operacao = input("por favor escolha uma das operações validas: ")
        
        numero_1 = int(input("digite o primeiro valor: "))
        numero_2 = int(input("digite o segundo valor: "))
        
        if operacao == '+':
            print(f"{numero_1} + {numero_2} = {numero_1 + numero_2}")
        
        elif operacao == '-':
            print(f"{numero_1} - {numero_2} = {numero_1 - numero_2}")
        
        elif operacao == '*':
            print(f"{numero_1} * {numero_2} = {numero_1 * numero_2}")
        
        elif operacao == '/':
            print(f"{numero_1} / {numero_2} = {numero_1 / numero_2}")
        
        elif operacao == '%':
            print(f"{numero_1} % {numero_2} = {numero_1 % numero_2}")
        
        elif operacao == '//':
            print(f"{numero_1} // {numero_2} = {numero_1 // numero_2}")

        elif operacao == '**':
            print(f"{numero_1} ** {numero_2} = {numero_1 ** numero_2}")


print("""
bem vindo ah calculadora master! um projeto de calculadora que oferece diversas calcluadoras
integrados numa só,aqui estão as calculadoras disponiveis:
1-calculadora padrão
""")
pergunta = int(input("escolha umas das calculadoras para usar: "))
calculo(pergunta)

File: calculadora.c
#include <stdio.h>
#include <string.h>
#include <math.h>

int main(){
    int value_one;
    int value_two;
    char resposta[10];
    
    printf("bem vindo a versão da caclulazinha em c ");
    printf("\ndigite o primeiro valor:");
    scanf("%d", &value_one);
    printf("digite o segundo valor:");
    scanf("%d", &value_two);
    printf("essas são as operaçoes disponiveis:\n");
    printf("+ = soma\n- = subtrair\n* = multiplicar\n");
    printf("/ = dividir\n‰ = modular\n** = potencia:");
    scanf("%s", resposta);
    
    if(strcmp(resposta, "+") == 0){
        int operacao = value_one + value_two;
        printf("aqui esta o resultado da operacao\n");
        printf("%d %s %d = %d", value_one, resposta, value_two, operacao);
        printf("\nfim do programa (⁠•⁠‿⁠•⁠)");
    } 
    
    else if(strcmp(resposta, "-") == 0){
        int operacao = value_one - value_two;
        printf("aqui esta o resultado da operacao\n");
        printf("%d %s %d = %d", value_one, resposta, value_two, operacao);
        printf("\nfim do programa (⁠•⁠‿⁠•⁠)");
    } 
    
    else if (strcmp(resposta, "*") == 0){
        int operacao = value_one * value_two;
        printf("aqui esta o resultado da operacao\n");
        printf("%d %s %d = %d", value_one, resposta, value_two, operacao);
        printf("\nfim do programa (⁠•⁠‿⁠•⁠)");
    } 
    
    else if (strcmp(resposta, "/") == 0){
        int operacao = value_one / value_two;
        printf("aqui esta o resultado da operacao\n");
        printf("%d %s %d = %d", value_one, resposta, value_two, operacao);
        printf("\nfim do programa (⁠•⁠‿⁠•⁠)");
    } 
    
    else if (strcmp(resposta, "%") == 0){
        int operacao = value_one % value_two;
        printf("aqui esta o resultado da operacao\n");
        printf("%d %s %d = %d", value_one, resposta, value_two, operacao);
        printf("\nfim do programa (⁠•⁠‿⁠•⁠)");
    }
    
    else if (strcmp(resposta, "**") == 0){
        int operacao = pow(value_one, value_two);
        printf("aqui esta o resultado da operacao\n");
        printf("%d %s %d = %d", value_one, resposta, value_two, operacao);
        printf("\nfim do programa (⁠•⁠‿⁠•⁠)");
    }

    return 0;
}